draft-bash-rfc7958bis.mkd

```diff
@@ -299,6 +300,8 @@ Figure 2 describes two fictitious trust anchors for the root zone.
 ~~~
 <?xml version="1.0" encoding="UTF-8"?>
 
+<!-- Note that these trust anchors are fictitious. -->
```

Comment:
Could we have examples of the comment at multiple locations in the XML?

That would make it more clear that the comments can occur anywhere. Developers shouldn't be writing their own XML parsers, but even if they aren't, they may have a switch case that fails when encountering a comment token at a random place in the document.

For this example, we could comment the two keys with:

`<!-- This key is no longer valid, since validUntil is in the past -->`

`<!-- This key may or may not be valid; an update to this file may add an validUntil tag -->`

Reply:
at some point this will deviate from the business of trust anchor publication into a remedial class in XML, but I don't think we are there yet. I agree that examples are good places to illustrate that comments might appear in crazy places.